database/bigquery/requirements.txt

```diff
@@ -0,0 +1,2 @@
+pandas==1.0.4
+google-cloud-bigquery-1.25.0
```

Comment:
Is this the correct syntax? Does Google put the version in the name of package?

Reply:
Fixed